\doxysection{Bad\+Command Class Reference}
\hypertarget{classpip_1_1__internal_1_1exceptions_1_1BadCommand}{}\label{classpip_1_1__internal_1_1exceptions_1_1BadCommand}\index{BadCommand@{BadCommand}}


Inheritance diagram for Bad\+Command\+:\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=166pt]{d1/d6e/classpip_1_1__internal_1_1exceptions_1_1BadCommand__inherit__graph}
\end{center}
\end{figure}


Collaboration diagram for Bad\+Command\+:\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=166pt]{d5/da7/classpip_1_1__internal_1_1exceptions_1_1BadCommand__coll__graph}
\end{center}
\end{figure}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Raised when virtualenv or a command is not found\end{DoxyVerb}
 

Definition at line \mbox{\hyperlink{__internal_2exceptions_8py_source_l00282}{282}} of file \mbox{\hyperlink{__internal_2exceptions_8py_source}{exceptions.\+py}}.



The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
/home/liwuen/projects\+\_\+dev/pqc-\/engineering-\/ssec-\/23/venv/lib/python3.\+12/site-\/packages/pip/\+\_\+internal/\mbox{\hyperlink{__internal_2exceptions_8py}{exceptions.\+py}}\end{DoxyCompactItemize}
